subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_lte_notification.c

```diff
@@ -191,6 +191,19 @@ void lte_notify_handler(const struct lte_lc_evt *const evt)
 		lwm2m_lte_reg_handler_notify(evt->nw_reg_status);
 		break;
 #endif /* CONFIG_LWM2M_CLIENT_UTILS_NWK_REG_NOTIFICATION */
+
+#if defined(CONFIG_LTE_LC_TAU_PRE_WARNING_NOTIFICATIONS)
+	case LTE_LC_EVT_TAU_PRE_WARNING:
+		struct lwm2m_ctx *ctx = lwm2m_rd_client_ctx();
+
+		if (!ctx) {
```

Comment:
At least on Zephyr side defining variables in the middle of a function is frowned upon. If we want to follow the same convention please define the variable at the beginning of the function or the following will also do:

```suggestion
		if (!lwm2m_rd_client_ctx()) {
```